flume-ng-sinks/flume-ng-hbase-sink/src/main/java/org/apache/flume/sink/hbase/HBaseSink.java

```diff
@@ -197,6 +198,11 @@ public void stop() {
   @SuppressWarnings("unchecked")
   @Override
   public void configure(Context context) {
+    if (!HBaseVersionCheck.hasVersionLessThan2(logger)) {
+      throw new ConfigurationException(
+          "HBase major version number must be less than 2 for hbase-sink. " +
+          "Use hbase2-sink instead.");
```

Comment:
Can this message be formulated in a way that it indicates hbase2-sink is not yet implemented?
Or is this PR work-in-progress, thus not yet to be merged?

Reply:
The second line of this message should be included in the commit where hbase2-sink is added. I will delete it from here.

Reply:
Thank you @mcsanady for that updated patch. @bessbd what do you think about the chage, are you ok with it? Thanks.

Reply:
@adenes : I haven't tried the new patch myself, but it looks good to me on a first read.